apps/web/src/components/root-app-sidebar/index.tsx

```diff
@@ -97,10 +98,22 @@ export const RootAppSidebar = ({
   }, [sidebarOpen]);
 
   const clientUpdateAvailable = useAtomValue(updateAvailableAtom);
+  const [history, setHistory] = useAtom(historyAtom);
+  const router = useMemo(() => {
+    return {
+      forward: () => {
```

Comment:
We may also want to add `allowForward` & `allowBackward` here that to be consumed by the buttons to control the usibility.